.gitignore

```diff
@@ -16,3 +16,5 @@ src/agent/protocols/src/*.rs
 build
 src/tools/log-parser/kata-log-parser
 tools/packaging/static-build/agent/install_libseccomp.sh
+.envrc
+.direnv
```

Comment:
I suggest these items should be added to your global git config. So, you don't have to add these over and over again, right? Frankly, I am okay with this PR.

```shell
$ git config --global core.excludesFile '.envrc'
$ git config --global core.excludesFile '.direnv'
```

Reply:
Fair enough. Lgtm!